jbpm-wb-executor-service/jbpm-wb-executor-service-client/src/main/java/org/jbpm/workbench/es/client/editors/quicknewjob/NewJobPresenter.java

```diff
@@ -0,0 +1,181 @@
+        view.show();
+    }
+
+    public void createJob(String jobName,
+                          Date dueDate,
+                          String jobType,
+                          String numberOfRetries,
+                          List<RequestParameterSummary> parameters) {
+        if (validateForm(jobName,
+                         jobType,
+                         numberOfRetries)) {
+            Map<String, String> jobCtxMap = new HashMap<String, String>();
+            if (parameters != null) {
+                for (RequestParameterSummary param : parameters) {
+                    jobCtxMap.put(param.getKey(),
+                                  param.getValue());
+                }
+            }
+            jobCtxMap.put(RequestDataSetConstants.COLUMN_RETRIES,
+                          numberOfRetries); // TODO make legacy keys hard to repeat by accident
+            jobCtxMap.put(RequestDataSetConstants.COLUMN_BUSINESSKEY,
+                          jobName); // TODO make legacy keys hard to repeat by accident
+
+            executorServices.call(
+                    (Long requestId) -> {
+                        displayNotification(constants.RequestScheduled(requestId));
+                        requestCreatedEvent.fire(new RequestChangedEvent(requestId));
+                        view.hide();
+                    },
+                    (Message message,
+                     Throwable throwable) -> {
+                        view.showBasicPane();
+                        if (isInvalidCommandTypeError(throwable)) {
+                            view.showInvalidTypeErrorMessage();
+                        } else {
+                            view.showInlineNotification(throwable.getCause().getMessage());
+                        }
+                        return false;
+                    }).scheduleRequest(serverTemplateId,
+                                       jobType,
+                                       dueDate,
+                                       jobCtxMap);
+        } else {
+            view.showBasicPane();
+        }
+    }
+
+    protected boolean validateForm(String jobName,
+                                   String jobType,
+                                   String jobRetries) {
+        boolean valid = true;
+        view.cleanErrorMessages();
+        if (jobName.isEmpty()) {
+            view.showEmptyNameErrorMessage();
+            valid = false;
+        }
+
+        if (jobType.isEmpty()) {
+            view.showEmptyTypeErrorMessage();
+            valid = false;
+        }
+
+        if (jobRetries.isEmpty()) {
+            view.showEmptyRetriesErrorMessage();
+            valid = false;
+        }
+
+        return valid;
+    }
+
+    private boolean isInvalidCommandTypeError(Throwable throwable) {
+        String message = "";
+        if (throwable.getMessage() != null) {
+            message = throwable.getMessage();
+        } else if (throwable.getCause() != null) {
```

Comment:
As I mentioned, here we have a potential _NullPointerException_ gap. The _else if_ expression should be expanded to:
```else if (throwable.getCause() != null && throwable.getCause().getMessage() != null)```